tests/trestle/core/remote/cache_test.py

```diff
@@ -87,15 +87,10 @@ def oscal_read_mock(*args, **kwargs):
     fetcher, _ = get_catalog_fetcher(tmp_trestle_dir)
     # mock bad read of oscal model
     monkeypatch.setattr(Catalog, 'oscal_read', oscal_read_mock)
-    with pytest.raises(err.TrestleError):
+    with pytest.raises(err.TrestleError, match='get_oscal failure'):
```

Comment:
My only problem with this is that error strings can change.

Reply:
I think that's ok since the test would fail in an obvious way.  The error strings should not change often.  It makes the tests slightly more brittle but provides confidence the test is failing in the intended way.